examples/queries/examples/query_deployed_bytecode.rs

```diff
@@ -0,0 +1,30 @@
+//! Example of querying deployed bytecode of a contract on Ethereum network.
+
+use alloy_network::Ethereum;
+use alloy_primitives::address;
+use alloy_provider::{HttpProvider, Provider};
+use alloy_rpc_client::RpcClient;
+use alloy_rpc_types::{BlockId, BlockNumberOrTag};
+use alloy_transport_http::Http;
+use eyre::Result;
+use reqwest::Client;
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    let provider = init();
+
+    // Get bytecode of USDC-ETH Uniswap V3 pool
+    let pool_address = address!("88e6A0c2dDD26FEEb64F039a2c41296FcB3f5640");
+
+    let bytecode =
+        provider.get_code_at(pool_address, BlockId::Number(BlockNumberOrTag::Latest)).await?;
+
+    println!("Bytecode: {:?}", bytecode);
+
+    Ok(())
+}
+
+fn init() -> HttpProvider<Ethereum> {
+    let http = Http::<Client>::new("https://eth.llamarpc.com".parse().unwrap());
+    HttpProvider::new(RpcClient::new(http, true))
+}
```

Comment:
Starting to wonder whether we should pull all these example calls in 1 file, with functions, and call all the functions one after the other, possibly making a more structured / informational example than just calling 1 method (and duplicating much code)

like this https://github.com/paradigmxyz/reth/blob/main/examples/db-access.rs

Reply:
Yeah that would make directory level organisation also simpler and could have all examples in one crate instead of separate crates for each example type.